hack/prr-baseline.txt

```diff
@@ -0,0 +1,372 @@
+# This file contains KEPs that are allowed to fail PRR validation
+# These are existing KEPs that were created before the requirement was enforced
+# New KEPs must pass validation - do not add new entries to this file manually
```

Comment:
So new KEPs are covered. But that's not really where we have problems, right? New KEPs copy the current template and thus typically have all section.

More interesting are old KEPs which need to be updated when extending the template. How will that be handled? It looks like extending the template and the list of required sections will cause verification failures for all KEPs not in this list.

Is the plan then to update the list? Then we don't solve the problem of ensuring that KEPs e.g. in alpha go to beta with an up-to-date PRR.